ingress/nginx.tmpl

```diff
@@ -0,0 +1,44 @@
+worker_processes  1;
```

Comment:
This needs to be configurable w/ a flag so we can specify at pod level (cpu request = worker_processes).